packages/next/export/index.js

```diff
@@ -129,17 +149,12 @@ export default async function (dir, options, configuration) {
   // Copy public directory
   if (existsSync(publicDir)) {
     log('  copying "public" directory')
-    await cp(
-      publicDir,
-      outDir,
-      {
-        expand: true,
-        filter (path) {
-          // Exclude paths used by pages
-          return !exportPathMap['/' + path]
-        }
+    await recursiveCopy(publicDir, outDir, {
+      filter (path) {
+        // Exclude paths used by pages
+        return !exportPathMap[path]
```

Comment:
@lucleray One of the tests for `/public` is not passing because of this, you sure the backslash is not required ?

Reply:
The `/` is not required here because the filter function is a bit different from the previous one.

See this test : https://github.com/zeit/next.js/blob/2e1862a279d4c2d5ec563f06a45a6de49dbbeb8f/test/unit/recursive-copy.test.js#L21-L35

I think the tests are not passing on windows, because pathes on windows are generated with `\` instead of `/`. If you have windows, you can check if it's the case by running :

```
yarn testonly test/unit/recursive-copy.test.js
```

This test shouldn't pass on windows (I can't check because unit tests don't seem to run on Windows in the test suite).

I'll fix that and run the test suite again to see if the integration tests pass.

Reply:
Attempted to fix this in https://github.com/zeit/next.js/pull/7263/commits/497c305ed213ab3e66f6e4c88baaa168df1bcfbb.

Reply:
@lucleray Mmm the previous implementation was always turning paths to `/` then ? shouldn't we do the same ? even for the returned result. btw I'm using Windows and locally all tests work, I guess this would be an exception

Reply:
Oh yes that's right, I got confused by `recursiveReadDir`, anyway this is great, once you fix the symlinks think I'm okay with it 👍

Reply:
I found the reason of the error. The symlink I was copying in the tests was created on a my mac computer and therefore was not working on windows (I didn't dive into details into this, but you can find some on the [wikipedia article](https://en.wikipedia.org/wiki/Symbolic_link)).

So, intead, I now create the "fixtures" for the test programmatically, so they will be created properly according to the os in use. All the tests are now passing ✅